### PR TITLE
feature/FOUR-18857:  FE: Implement the new collapse like Launchpad

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
FOUR-18498: FE: Implement the new collapse like Launchpad

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-18857

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-e78922b413.engk8s.processmaker.net

File: resources/views/requests/show.blade.php [hunk 1]
```diff
@@ -22,454 +22,450 @@
...
-                  @click="switchTab('summary')" aria-selected="false"
-                  v-bind:class="{ 'nav-link':true, active: showSummary && !activePending }">
-                  {{ __('Summary') }}
-                </a>
-              </li>
-              @if ($request->status === 'COMPLETED' && !$request->errors)
-                @can('editData', $request)
+  <div id="request">
+    <div class="menu-mask" :class="{ 'menu-open': showMenu }"></div>
+    <div class="process-catalog-main" :class="{ 'menu-open': showMenu }">
```
Comment (fagubla) [inline resources/views/requests/show.blade.php:27]: process-catalog-main?

File: resources/views/tasks/edit.blade.php [hunk 2]
```diff
@@ -27,341 +27,335 @@ function() use ($task) {
...
-                                          :timeline="false"
-                                          />
-                          </div>
-                        </div>
-                        @can('editData', $task->processRequest)
-                            <div v-if="task.process_request.status === 'ACTIVE'" id="tab-data" role="tabpanel" aria-labelledby="tab-data" class="card card-body border-top-0 tab-pane p-3">
-                                @include('tasks.editdata')
+  <div id="task">
+    <div class="menu-mask" :class="{ 'menu-open': showMenu }"></div>
+    <div class="process-catalog-main" :class="{ 'menu-open': showMenu }">
```
Comment (fagubla) [inline resources/views/tasks/edit.blade.php:32]: same here

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-e78922b413.engk8s.processmaker.net

**Pull request merged**